// check_error 2:28 "mismatching return type 'int', expected 'void'"
func foo(a: int, b: int) { return a * b; }
